lib/migration-payloads/validation/errors.js

```diff
@@ -39,6 +39,17 @@ const errors = {
     },
     NO_TYPE_CHANGE: (fieldId, ctId, parentFieldType, fieldType) => {
       return `Cannot change the type of field "${fieldId}" on content type "${ctId}" from "${parentFieldType}" to "${fieldType}". Field types cannot be changed.`;
+    },
+    validations: {
+      DUPLICATED_VALIDATION: (duplicatedValue) => {
+        return `A field can't have duplicates in the validations array. Duplicated "${JSON.stringify(duplicatedValue)}"`;
```

Comment:
I'd changes this to "Duplicate: "${JSON.stringify(duplicatedValue)}"

Reply:
But then you don't know the context of where and why there is a duplicate 🙁

Reply:
Oh I only meant change "Duplicated" to "Duplicate:"